org.hlwd.biblemultithelife.yml

```diff
@@ -0,0 +1,51 @@
+app-id: org.hlwd.biblemultithelife
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: biblemultithelife
+separate-locales: false
+finish-args:
+  - --socket=x11
+  - --socket=fallback-x11
```

Comment:
it's either `x11` alone or `wayland` + `fallback-x11`. Not both

Reply:
Ok, solved